13-1-1/update_indicator.R

```diff
@@ -0,0 +1,387 @@
+### 13-1-1 ###
+
+# Output table gives number of deaths by cause of death, by sex,
+# by country, by year (2001 to present).
+# It also gives Age-Standardised mortality rates by Sex for 2013 to present.
+
+
+# download and read in data ----------------------------------------------------
+nomis_deaths <- read.csv(nomis_disaster_deaths_link) %>%
+  mutate(across(where(is.factor), as.character)) %>%
+  mutate(across(where(is.character), str_to_sentence)) %>% 
+  mutate(across(where(is.character), str_squish)) 
+
+nomis_mortality <- read.csv(nomis_mortality_link) %>% 
+  mutate(across(where(is.factor), as.character)) %>% 
+  mutate(across(where(is.character), str_to_sentence)) %>% 
+  mutate(across(where(is.character), str_squish)) 
+
+ons_deaths1 <- openxlsx::read.xlsx(ons_disaster_death_link, sheet = ons1, colNames = FALSE) %>%
```

Comment:
ons_deaths is a bit meaningless - ALL the data including the nomis stuff could be described as ons deaths, though i get the wish to distingush between nomis and ons website data. If you put the nomis and historic stuff in separate scripts you could lose the need to distinguish them in the names here.
A more meaningful name for ons_deaths1 might  then be e.g. headline_df